### ISSUE TITLE
After install error on linting is generated

### ISSUE THREAD

**Issue by gabrielstuff** [eddyerburgh/parcel-vuejs-template#2]
The following error thrown after install with yarn and eslint standard.

```
$ eslint --ext .js,.vue src --fix
Cannot read config file: /curator/.eslintrc
Error: end of the stream or a document separator is expected at line 4, column 7:
      root: true,
          ^
YAMLException: Cannot read config file: /curator/.eslintrc
Error: end of the stream or a document separator is expected at line 4, column 7:
      root: true,
```

**Comment by andycopley** [comment]
Rename `.eslintrc` to `.eslintrc.js` to force ESLint to read the configuration file as Javascript and not YAML.

**Comment by eddyerburgh** [comment]
Fixed with #3
